### PR TITLE
Implement basic user switching

### PR DESCRIPTION
**Pull request by jonasdekeukelaere**
Implement a basic working of user switching, only available for admins.

The link (icon) to switch to an user is shown in the user overview. To switch back use the link replacing the logout button in the top right menu.

### CODE REVIEW

File: src/SumoCoders/FrameworkUserBundle/Entity/User.php [hunk 1]
```diff
@@ -26,4 +26,9 @@ public function getRoles(): array
     {
         return ['ROLE_USER'];
     }
+
+    public function canSwitchTo(BaseUser $user): bool
```
Comment (StijnVrolijk) [inline src/SumoCoders/FrameworkUserBundle/Entity/User.php]: wouldn't this be a good default to be placed in the MultiUserBundle itself?

Comment (jonasdekeukelaere) [reply src/SumoCoders/FrameworkUserBundle/Entity/User.php]: Maybe yes, I have a look into this, just in BaseUser?

Comment (jonasdekeukelaere) [reply src/SumoCoders/FrameworkUserBundle/Entity/User.php]: @StijnVrolijk https://github.com/sumocoders/FrameworkMultiUserBundle/pull/65

Comment (jonasdekeukelaere) [reply src/SumoCoders/FrameworkUserBundle/Entity/User.php]: @StijnVrolijk Fixed 3fdb4d3

File: src/SumoCoders/FrameworkUserBundle/Resources/translations/messages.nl.yml [hunk 2]
```diff
@@ -17,6 +17,7 @@ user:
       admin: beheerder toevoegen
     settings: instellingen
     logout: uitloggen
+    switchBack: naar originele gebruiker
```
Comment (carakas) [inline src/SumoCoders/FrameworkUserBundle/Resources/translations/messages.nl.yml]: don't we have english translations as well?

Comment (jonasdekeukelaere) [reply src/SumoCoders/FrameworkUserBundle/Resources/translations/messages.nl.yml]: Fixed 4df5cef

**Pull request merged**